src/triage/experiments/base.py

```diff
@@ -454,9 +482,11 @@ def master_feature_dictionary(self):
             ),
         )
         logging.info("Computed master feature dictionary: %s", result)
+        with self.checkout_experiment_for_update() as experiment:
+            experiment.total_features = len(list(feature for table_features in result.values() for feature in table_features))
```

Comment:
or, instead of building a list, something like 🤷‍♂️ 

```suggestion
            experiment.total_features = sum(1 for _feature in itertools.chain.from_iterable(result.values()))
```